src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,21 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+public enum ColorSupplier {
+    RED,
+    GREEN,
+    YELLOW,
+    PURPLE,
+    BROWN,
+    BLUE,
+    GRAY,
+    ORANGE;
+
+    private static final Random RANDOM = new Random();
+
+    public static String getRandomColor() {
```

Comment:
**Don’t use static methods in your solution**
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-use-static-methods-in-your-solution)